feat: new endpoints added

saledjenic opened the pull request.

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (12)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | d30528de | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3856/1/) | 2023-08-07 06:40:58 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230807-063748-d30528-pr3856.zip) |
| :heavy_check_mark: | d30528de | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3856/1/) | 2023-08-07 06:42:21 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230807-063743-d30528-pr3856.aar) |
| :heavy_check_mark: | d30528de | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3856/1/) | 2023-08-07 06:42:57 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230807-063748-d30528-pr3856.zip) |
| :heavy_check_mark: | d30528de | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3856/1/) | 2023-08-07 07:02:49 | ~25 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3856/1//consoleText) |
| | | | | | | |
| :heavy_check_mark: | ea202c52 | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3856/2/) | 2023-08-07 13:40:04 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230807-133821-ea202c-pr3856.aar) |
| :heavy_check_mark: | ea202c52 | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3856/2/) | 2023-08-07 13:42:19 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230807-133821-ea202c-pr3856.zip) |
| :heavy_check_mark: | ea202c52 | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3856/2/) | 2023-08-07 13:43:51 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230807-133821-ea202c-pr3856.zip) |
| :heavy_check_mark: | ea202c52 | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3856/2/) | 2023-08-07 14:03:08 | ~24 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3856/2//consoleText) |
| | | | | | | |
| :heavy_check_mark: | afff6f35 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3856/3/) | 2023-08-07 15:04:57 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230807-150342-afff6f-pr3856.zip) |
| :heavy_check_mark: | afff6f35 | [#3](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3856/3/) | 2023-08-07 15:05:05 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230807-150342-afff6f-pr3856.aar) |
| :heavy_check_mark: | afff6f35 | [#3](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3856/3/) | 2023-08-07 15:06:24 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230807-150342-afff6f-pr3856.zip) |
| :heavy_check_mark: | afff6f35 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3856/3/) | 2023-08-07 15:23:07 | ~19 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3856/3//consoleText) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 2d582d16 | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3856/4/) | 2023-08-07 15:48:21 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230807-154657-2d582d-pr3856.zip) |
| :heavy_check_mark: | 2d582d16 | [#4](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3856/4/) | 2023-08-07 15:48:47 | ~1 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230807-154656-2d582d-pr3856.aar) |
| :heavy_check_mark: | 2d582d16 | [#4](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3856/4/) | 2023-08-07 15:50:27 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230807-154656-2d582d-pr3856.zip) |
| :heavy_multiplication_x: | 2d582d16 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3856/4/) | 2023-08-07 15:52:20 | ~5 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3856/4//consoleText) |
| | | | | | | |
| :heavy_check_mark: | ab1134bb | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-3856/5/) | 2023-08-08 11:43:22 | ~1 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-230808-114208-ab1134-pr3856.zip) |
| :heavy_check_mark: | ab1134bb | [#5](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-3856/5/) | 2023-08-08 11:44:24 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-230808-114204-ab1134-pr3856.aar) |
| :heavy_check_mark: | ab1134bb | [#5](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-3856/5/) | 2023-08-08 11:46:00 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-230808-114204-ab1134-pr3856.zip) |
| :heavy_check_mark: | ab1134bb | [#5](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3856/5/) | 2023-08-08 12:04:08 | ~21 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-3856/5//consoleText) |

saledjenic committed:
feat: new endpoints added

- `HasPairedDevices`
- `MakePrivateKeyKeypairFullyOperable`
- `MakeSeedPhraseKeypairFullyOperable`

qfrank reviewed (approved):
LGTM
